src/auth/enums/userRole.enum.ts

```
@@ -0,0 +1,4 @@
+export enum UserRole {
+    FREELANCER = 'freelancer',
```

ESLint

[error] Delete `··`

(prettier/prettier)